src/js/utils/background.js

```diff
@@ -159,8 +165,17 @@ export const backgroundAndTextColors = (backgroundArg, textArg, theme) => {
   } else {
     backgroundColor = normalizeBackgroundColor(background, theme);
     const shade = darkContext(backgroundColor, theme);
+    let transparent;
+
+    if (backgroundColor && canExtractRGBArray(backgroundColor)) {
+      const colorArray = getRGBArray(backgroundColor);
+      // check if the alpha value is less than 0.5
+      if (colorArray[3] < 0.5) transparent = true;
```

Comment:
Further up in this function we use `0.3` to determine if a background is translucent. Should we align or is this intentionally different?

Reply:
The section above that has `0.3` is used when we define `background.color` in the theme and it looks like there are some different issues with the code in this section. For example if a background color has vey little transparency it doesn't take that into account and shows white background & white text, so that might be a separate issue to look into.

We could change `colorArray[3] < 0.5` to `colorArray[3] < 0.3`, this will affect the transpancy level we flip the text color at. However I think `0.5` works better than `0.3`

Here is an example where we have `colorArray[3] < 0.3`
<img width="167" alt="Screen Shot 2023-02-13 at 12 28 50 PM" src="https://user-images.githubusercontent.com/54560994/218563808-b6b42a74-73de-4ce1-bfc3-3bc666c66d02.png">


and here is the same example with `colorArray[3] < 0.5`
<img width="154" alt="Screen Shot 2023-02-13 at 12 29 17 PM" src="https://user-images.githubusercontent.com/54560994/218563833-94bcb495-c675-4bef-9895-5150f13eb607.png">